nixos/modules/services/backup/borgbackup.nix

```diff
@@ -82,12 +82,13 @@ let
         # Only run when no other process is using CPU or disk
         CPUSchedulingPolicy = "idle";
         IOSchedulingClass = "idle";
-        ProtectSystem = "strict";
+        ProtectSystem = (if cfg.privateTmp then "strict" else "full");
```

Comment:
This can be left `strict` if you add `/tmp` to `ReadWritePaths`.

Reply:
Indeed, didn't even think of that.